docs/heuristics.rst

```diff
@@ -86,6 +86,19 @@ or::
         ...
         return seqinfos  # ordered dict containing seqinfo objects: list of DICOMs
 
+---------------------------------------------------------------
+``custom_seqinfo(series_files, wrapper)``
+---------------------------------------------------------------
+If present this function will be called on eacg group of dicoms with
```

Comment:
```suggestion
If present this function will be called on each group of dicoms with
```

we have `codespell` workflow which should pickup typos. If you rebase -- we better see it working on this PR too (for some reason it did not even though I think it was back 2 weeks ago when you started this PR)

Reply:
oh, this is PR to my PR -- I will take it on and then rebase my PR. Thanks!